controllers/get-documentation.controller.js

```diff
@@ -0,0 +1,5 @@
+
+exports.getDocumentation = (request, response) => {
+    const documentation = require('../endpoints.json')
+    response.status(200).send(documentation)
+}
```

Comment:
Really nice controller for this and I agree with the lack of a model function for this as we are not querying the database so we dont really need one. Nice